src/modules/trading/components/trading--wrapper/trading--wrapper.jsx

```diff
@@ -410,23 +414,48 @@ class TradingWrapper extends Component {
             type={selectedNav}
             action={e => {
               e.preventDefault();
-              onSubmitPlaceTrade(
-                market.id,
-                selectedOutcome.id,
-                s.trade,
-                s.doNotCreateOrders,
-                (err, tradeGroupID) => {
-                  // onSent/onFailed CB
-                  if (!err) {
-                    this.clearOrderForm();
+              if (!marketReviewTradeSeen) {
+                marketReviewTradeModal({
+                  marketId: market.id,
+                  cb: () => {
+                    onSubmitPlaceTrade(
```

Comment:
let's pull out `onSubmitPlaceTrade` method so it's not duplicated. put in it's own method in the component.

Reply:
will do